src/ast/mod.rs

```diff
@@ -280,7 +280,23 @@ impl fmt::Display for Expr {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             Expr::Identifier(s) => write!(f, "{}", s),
-            Expr::MapAccess { column, key } => write!(f, "{}[\"{}\"]", column, key),
+            Expr::MapAccess { column, keys } => {
+                write!(
+                    f,
+                    "{}{}",
+                    column,
+                    keys.iter()
+                        .map(|k| {
+                            match k {
+                                k @ Value::Number(_, _) => format!("[{}]", k),
+                                Value::SingleQuotedString(s) => format!("[\"{}\"]", s),
```

Comment:
It is odd to me that  a `SingleQuotedString` is emitted with double quotes (`"`) but it is consistent with the existing implementation